service-seed-without-router/play-service/pom.xml

```diff
@@ -22,63 +32,94 @@
             </snapshots>
         </pluginRepository>
     </pluginRepositories>
+
     <dependencies>
+        <!-- Don't move this dependency below play dependency otherwise it will
+            be overriden by play jackson jar, which uses older version -->
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${jackson.version}</version>
+            <version>2.9.5</version>
```

Comment:
change the property value in parent pom and use property key's from parent pom

Reply:
done